wremnants/datasets/datagroups.py

```
@@ -227,29 +218,51 @@ def copyWithAction(self, action, name, refproc, refname, label, color):
         )
         self.groups[name][refname] = action(self.groups[refproc][refname])
 
+    def setSelectOp(self, op, processes=None, exclude=False): 
+        if processes == None:
+            procs = self.getDatagroups()
+        else:
+            if exclude:
+                procs = self.getDatagroups(excluded_procs=processes)
+            else:
+                procs = [processes] if isinstance(processes, str) else [p for p in processes]
+        for proc in procs:
+            if proc not in self.groups.keys():
+                logger.warning(f"In setSelectOp(): process {proc} not found")
```

Throw an exception rather than warning+quit